plugins/woocommerce/client/admin/client/homescreen/activity-panel/orders/index.js

```diff
@@ -281,6 +261,9 @@ function OrdersPanel( { unreadOrdersCount, orderStatuses } ) {
 			return { isRequesting: false };
 		}
 
+		/* eslint-disable @wordpress/no-unused-vars-before-return */
+		const { getCurrencySymbols } = select( COUNTRIES_STORE_NAME );
```

Comment:
I think you could move this line a little bit later after the first return statement so we don't need to use `/* eslint-disable @wordpress/no-unused-vars-before-return */`.  🙂

Reply:
Thank you! Fixed it in [Fix lint issue](https://github.com/woocommerce/woocommerce/pull/55464/commits/28c49adeefaa63d3f1d34ba31279a06289737cc7)